### ISSUE TITLE
Need details for purchase

### ISSUE THREAD

**Issue by padmarajm** [dearhive/dearflip-jquery-flipbook#50]
We are interested in buying dearflip JQuery version. We have some questions before that. On website I could not find a way to post questions. Is there some way to reach out to the author for further discussion

**Comment by padmarajm** [comment]
Sure. Here are my questions:

1. Can you share the end user license?
2. You have license per site. so does qa.abc.com, abc.com, staging.abc.com count as 1 site?
3. We have a requirement to disable download, copy, print screen, print options on the e-reader, both on desktop and mobile. Is this possible?
4. Can you share the latest vulnerability report of the code?

**Comment by padmarajm** [comment]
ok sure. do you think we can customize the code once we buy to handle these features? also can you please share details on other points?

**Comment by deepak-ghimire** [comment]
Very less likely with our codebase.We use pdf.js in the core. Due to that it's out of question.

Please look elsewhere.

**Comment by padmarajm** [comment]
Hi Deepak,
We noticed you do have option to disable download. mentioned here: https://dearflip.com/docs/dearflip-jquery/getting-started/#options-and-advanced-usage

Would it be possible to share licensing details and also vulnerability report?

**Comment by deepak-ghimire** [comment]
That is just hiding the download button. Your pdf file link will be visible in source and can be downloaded too.
Then again print screen and others? They cannot be prevented.

**Comment by padmarajm** [comment]
Yes, we are trying to check with our business team. We do find your reader to be really good and want to proceed. Can you share the licensing and vulnerability report?

**Comment by padmarajm** [comment]
Basically if it is npm module you get to see the vulnerabilies of any library you use, for example JQuery version.... 
Can you share the License terms?

**Comment by padmarajm** [comment]
thanks @deepak-ghimire . One more question. we have qa.abc.com, staging.abc.com and abc.com. Can I buy 1 site or do I need to buy 1 license for each of these subdomains?

**Comment by deepak-ghimire** [comment]
What is the purpose? If you can clarify I can suggest accordingly.

Staging sites do not consume licenses.

**Comment by padmarajm** [comment]
Thanks @deepak-ghimire . We have different environments to test as we progress our code. First is where our testing team tests, then we have 1 for User acceptance testing.. we use same domain but use different subdomain for each of these environments. like qa.abc.com, staging.abc.com...

So 1 license should suffice right?

**Comment by padmarajm** [comment]
While reading thru the license terms it mentions:
"_Single Website: Refers to one unique domain name (e.g., “example.com”) registered on the internet. Subdomains (e.g., “blog.example.com”) and individual sites within a WordPress Multisite environment are considered separate and distinct sites for the purpose of this license. Each requires its own individual license._"

So for my scenario do I need to buy 1 license each for qa.example.com , staging.example.com, www.example.com? Or will 1 license be used across all of them?

**Comment by deepak-ghimire** [comment]
How many will be there? 
Apart from staging/test subdomain, you will need extra license.

www.example.com one license
qa.example.com requires extra license,
staging.example.com not required extra license

For test environments naming is not an issues, you can keep any following the rules.

testQA.example.com - valid 
qa.example.com - invalid

So if you are using staging/test subdomain, follow the naming rules that denote that they are test sites.

TLDs that are considered as dev or staging:
*.dev
*.test
*.local
*.staging
*.example
*.invalid

Subdomains that are considered as dev or staging:
local.*
dev*.*
test*.*
stage*.*
staging*.*
stagingN*.*

Additionally, if your domain is localhost (with any port), it will also be treated as a localhost domain.

**Comment by padmarajm** [comment]
@deepak-ghimire  you had mentioned youa re using JQuery > 1.2. But when we check the demo page here: https://dearflip.com/pdf-viewer/ , it seems to be using 3.7 version which is latest. can you please confirm that is the version being used in the paid version?

**Comment by deepak-ghimire** [comment]
Hi,
We use jquery >1
That means 3.5 is also greater tha 1.2

Our live site uses wordpress and it has jquery 3.5
So you can add your version of jQuery.

Is there a problem with jquery version 3.5?
